win32/src/win32evtlog.i

```diff
@@ -562,8 +562,7 @@ static PyObject *FormatMessageInternal(EVT_HANDLE metadata, EVT_HANDLE event, DW
 		allocated_size += 1; // +1 to double terminate the keyword list
 	}
 
-	allocated_size *= sizeof(WCHAR);
-	buf = (WCHAR *)malloc(allocated_size);
+	buf = (WCHAR *)malloc(allocated_size * sizeof(WCHAR));
```

Comment:
why this change? `allocated_size` is no longer the allocated size.

Reply:
Yes, I understand my mistake now. After my changes, the `allocated_size` had a misleading name and there was no need for these changes in `allocated_size` and malloc at all. So I'll return back this part of the code as it was.